motes.md

```diff
@@ -0,0 +1,42 @@
+* `BLS_WITHDRAWAL_PREFIX`
+  * Why int rather than bytes?
+* `MIN_SEED_LOOKAHEAD`
+  * Is this actually tunable?
+  * If so, what are the reprecussions?
+* `ACTIVATION_EXIT_DELAY`
+  * Reaquaint with purpose
```

Comment:
The problem is that attackers can know the randomness beyond `MIN_SEED_LOOKAHEAD`, and hence bias the shuffling by strategically registering/deregistering validators in the window between `MIN_SEED_LOOKAHEAD` and `MIN_SEED_LOOKAHEAD + 3` to "grind" the shuffling. (The number 3 is large enough to guarantee that, with high probability, an attacker's lookahead cannot go beyond `MIN_SEED_LOOKAHEAD + 3`.)